cmd/cvp-trigger/main.go

```diff
@@ -136,13 +265,64 @@ func main() {
 			if !ok {
 				logrus.WithField("object-type", fmt.Sprintf("%T", event.Object)).Fatal("received an unexpected object from Watch")
 			}
+			spec := downwardapi.NewJobSpec(prowJob.Spec, prowJob.Status.BuildID, prowJob.Name)
```

Comment:
Do we have a unit test for all this logic?

Reply:
We do now.